find_replace_macro.py

```diff
@@ -33,7 +33,8 @@ def walk_matches():
                                 "binzz",
                                 "dgcd",
                                 "dlcm",
-                                "equ"]:
+                                "equ",
+                                "fplus"]:
```

Comment:
I forgot to commit this separately. But here I'm adding fplus to the exclude list.